src/MoviePreview/MoviePreview.scss

```diff
@@ -14,14 +14,26 @@
 }
 
 .description-wrapper {
+  width: 80%;
   height: auto;
-  width: 50em;
-  background: rgba(253, 255, 252, 0.7);
   display: flex;
   align-items: center;
   position: relative;
 }
 
+.preview-react-player {
+  margin: 1em 0 5em 0;
+}
+
+.preview-wrapper {
+  margin: 3em;
+  position: relative;
+  background: rgba(253, 255, 252, 0.7);
```

Comment:
I think we would need to refactor some styles and create additional variables for colors and repetitive values later